appendices/boilerplate.txt

```diff
@@ -160,7 +160,7 @@ version of the header was used during their generation.
 Applications should not use this value as their
 slink:VkApplicationInfo::pname:apiVersion.
 Instead applications should explicitly select a specific fixed major/minor
-API version using, for example, one of the dname:VK_API_VERSION_*_* values.
```

Comment:
What is the purpose of this change? `dname:` seems perfectly appropriate here, doesn't it?

Reply:
dname currently does not handle wildcards *